content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14.md

```diff
@@ -0,0 +1,380 @@
+    [bootstrap-token] configured RBAC rules to allow the csrapprover controller automatically approve CSRs from a Node Bootstrap Token
+    [bootstrap-token] configured RBAC rules to allow certificate rotation for all node client certificates in the cluster
+    [addons] Applied essential addon: CoreDNS
+    [addons] Applied essential addon: kube-proxy
+
+    [upgrade/successful] SUCCESS! Your cluster was upgraded to "v1.14.0". Enjoy!
+
+    [upgrade/kubelet] Now that your control plane is upgraded, please proceed with upgrading your kubelets if you haven't already done so.
+    ```
+
+1.  Manually upgrade your CNI provider plugin.
+
+    Your Container Network Interface (CNI) provider may have its own upgrade instructions to follow.
+    Check the [addons](/docs/concepts/cluster-administration/addons/) page to
+    find your CNI provider and see whether additional upgrade steps are required.
+
+1.  Upgrade the kubelet and kubectl on the control plane node:
+
+    {{< tabs name="k8s_install_kubelet" >}}
+    {{% tab name="Ubuntu, Debian or HypriotOS" %}}
+    # replace x in 1.14.x-00 with the latest patch version
+    apt-mark unhold kubelet && \
+    apt-get update && apt-get install -y kubelet=1.14.x-00 kubectl=1.14.x-00 && \
+    apt-mark hold kubelet
+    {{% /tab %}}
+    {{% tab name="CentOS, RHEL or Fedora" %}}
+    # replace x in 1.14.x-0 with the latest patch version
+    yum install -y kubelet-1.14.x-0 kubectl-1.14.x-0 --disableexcludes=kubernetes
+    {{% /tab %}}
+    {{< /tabs >}}
+
+1. Restart the kubelet
+
+    ```shell
+    sudo systemctl restart kubelet
+    ```
+
+## Upgrade additional control plane nodes
+
+1.  Same as the first control plane node but use:
+
+```
+sudo kubeadm upgrade node experimental-control-plane
+```
+
+instead of:
+
+```
+sudo kubeadm upgrade apply
+```
+
+Also `sudo kubeadm upgrade plan` is not needed.
+
+## Ugrade worker nodes
+
+The upgrade procedure on worker nodes should be executed one node at a time or few nodes at a time,
+without compromising the minimum required capacity for running your workloads.
+
+### Upgrade kubeadm
+
+1.  Upgrade kubeadm on all worker nodes:
+
+    {{< tabs name="k8s_install_kubeadm_worker_nodes" >}}
+    {{% tab name="Ubuntu, Debian or HypriotOS" %}}
+    # replace x in 1.14.x-00 with the latest patch version
+    apt-mark unhold kubeadm && \
+    apt-get update && apt-get install -y kubeadm=1.14.x-00 && \
+    apt-mark hold kubeadm
+    {{% /tab %}}
+    {{% tab name="CentOS, RHEL or Fedora" %}}
+    # replace x in 1.14.x-0 with the latest patch version
+    yum install -y kubeadm-1.14.x-0 --disableexcludes=kubernetes
+    {{% /tab %}}
+    {{< /tabs >}}
+
+### Cordon the node
+
+1.  Prepare the node for maintenance by marking it unschedulable and evicting the workloads. Run:
+
+    ```shell
+    kubectl drain $NODE --ignore-daemonsets
+    ```
+
```

Comment:
can you please add `The output is similar to this:` or `You should see output similar to this:` how you used already in this doc? this should align with style guidelines

Reply:
@neolit123 fyi ^^

Reply:
updated.